gammapy/maps/wcs.py

```diff
@@ -913,9 +915,15 @@ def cutout(self, position, width, mode="trim", odd_npix=False):
         """
         width = _check_width(width) * u.deg
 
+        binsz = self.pixel_scales
+        width_npix = (width / binsz).to_value("")
+
+        invalid_size = width_npix<1
+        if np.any(invalid_size):
```

Comment:
Looking at the implementation I'm wondering whether we actually need the log statement. Maybe simplify the implementation to basically one line with `np.clip()` and document the behaviour in the docstring instead? I think the clipping behaviour is very reasonable and without alternative...so it maybe does not justify emitting a warning.